src/components/MessageHTMLBody.vue

```diff
@@ -3,9 +3,16 @@
 		<MdnRequest :message="message" />
 		<div v-if="hasBlockedContent" id="mail-message-has-blocked-content" style="color: #000000">
 			{{ t('mail', 'The images have been blocked to protect your privacy.') }}
-			<button @click="onShowBlockedContent">
-				{{ t('mail', 'Show images from this sender') }}
-			</button>
+			<Actions default-icon="icon-toggle">
+				<ActionButton icon="icon-toggle"
+					@click="onShowBlockedContent">
+					{{ t('mail', 'Show images from {sender}', {sender: message.from[0].email}) }}
```

Comment:
what if we rename to *Always show …* to put some emphasis on the fact that this is a permanent setting?